<launch>
    <include file="$(find object_recognition_cnn)/object_recognition_cnn.launch" />
    
	<node pkg="object_localization_estimator" type="object_localization_estimator_node" name="object_localization_estimator_node" output="screen">
        <!-- Topics that are tracked -->
        <param name="worldFrame" value="world" />
        <param name="cameraFrame" value="usbcam" />
        <param name="copterFrame" value="base_link" />
        <param name="topic_cnn" value="/detected_objects" />
        <param name="topic_publish_objects" value="/localized_objects" />
        <param name="topic_publish_walls" value="/localized_walls" />

        <!-- parameters of the camera -->
        <param name="fov_h" value="41" />
        <param name="fov_v" value="31" />
	</node>
	
</launch>

